### PR TITLE
Upgrade and replace web server with EmbedIO

### PR DESCRIPTION
**Pull request by YoRyan**
Replace the home-rolled web server with EmbedIO, separate and clean up the client webpages, and add a copy web content step to the build process. Also see [these changes](https://github.com/YoRyan/openrails/pull/1#issue-426759304) by Chris Jakeman.

Blueprints: 
[Foundation for interactive content on secondary screens/devices](https://blueprints.launchpad.net/or/+spec/secondary-screen-base)
[Use EmbedIO in place of in-house web-server](https://blueprints.launchpad.net/or/+spec/embedio-for-web-server)
[Organise web content](https://blueprints.launchpad.net/or/+spec/organise-web-content)

### CODE REVIEW

**Comment by YoRyan** [comment]
AppVeyor build has been fixed!

File: .appveyor.yml [hunk 1]
```diff
@@ -8,6 +8,8 @@ install:
     (New-Object Net.WebClient).DownloadFile('https://github.com/electron/rcedit/releases/download/v1.1.1/rcedit-x86.exe', 'rcedit-x86.exe')
 
     $env:PATH="$env:PATH;C:\lazarus;$((Get-ChildItem -Recurse C:\lazarus\fpc\strip.exe).DirectoryName);C:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\bin"
+before_build:
+- nuget restore Source\ORTS.sln
```
Comment (twpol) [inline .appveyor.yml:12]: Sorry, this change is not in line with how we build Open Rails - you should make all modifications within `Build.cmd` alone.

**Comment by cjakeman** [comment]
Closed this PR until we can find the right way to change build.cmd so that the Jenkins build works as well as the Appveyor one.

**Comment by YoRyan** [comment]
Replaced by #216.

**Pull request closed without merging**